project/Version.scala

```diff
@@ -4,12 +4,12 @@ object Version {
   val Scala213   = "2.13.8"
   val ScalaDotty = "3.1.0"
 
-  val zio       = "2.0.0-RC1"
-  val uzhttp    = "0.3.0-RC1"
+  val zio       = "2.0.0-RC2"
+  val zioHttp   = "2.0.0-RC3+1-7c2f06c7-SNAPSHOT"
```

Comment:
There was a bug in zio-http 2.0.0 RC3. They fixed it after I reported it. But it's only in a snapshot. Will update this once a proper RC is released.